src/NuGet.Commands/RestoreCommand.cs

```diff
@@ -149,7 +149,7 @@ public Task<RestoreResult> ExecuteAsync()
         {
             if (_request.Project.TargetFrameworks.Count == 0)
             {
-                _log.LogError(Strings.Log_ProjectDoesNotSpecifyTargetFrameworks);
+                _log.LogError(Strings.FormatLog_ProjectDoesNotSpecifyTargetFrameworks(_request.Project.Name));
```

Comment:
lowercase s on string as per the convention. 

What is doing the formatting here? is it using the current culture?